docs/jsonapi-spec.md

```diff
@@ -464,6 +465,42 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
  
 If an error occurs the command will not return `status`.
 
+
+### createCollection Command
+
+Creates a new collection in the current namespace.
+
+*Sample*
+
+```json
+{
+  "createCollection": {
+    "name": "purchase"
+  }
+}
+```
+
+#### createCollection Command Options
+
+The `createCollection` command does not support any options.
+
+#### createCollection Multi Document Failure Modes
+
+Fail Fast, a storage failure causes the command to stop processing.
+
+See [Multi-Document Failure Considerations](#multi-document-failure-considerations).
+
+#### createCollection Command Response
+
+| Response Elements | Description                                             |
+| ----------------- | ------------------------------------------------------- |
+| `data`            | Not present                                             |
+| `status`          | Preset with fields: `count: <zero-or-positive-integer>` |
```

Comment:
Has this been changed: to me it seems to return JSON Object like:

```
{
  "status": {
    "ok": 1
  }
}
```

Reply:
Thanks, updated in latest commit.